Assets/Editor/PathTestingEditor.cs

```diff
@@ -1,18 +1,28 @@
 ﻿using UnityEngine;
 using UnityEditor;
 
-[CustomEditor(typeof(PathTesting))]
+[CustomEditor(typeof(ubv.server.testing.PathTesting))]
 public class PathTestingEditor : Editor
 {
     public override void OnInspectorGUI()
     {
         DrawDefaultInspector();
 
-        PathTesting myTarget = (PathTesting)target;
+        ubv.server.testing.PathTesting myTarget = (ubv.server.testing.PathTesting)target;
 
         if(GUILayout.Button("Test Random Path"))
         {
             myTarget.TestRandomPath();
         }
+
+        if (GUILayout.Button("Free node"))
+        {
+            myTarget.FreeNode();
+        }
+
+        if (GUILayout.Button("Block node"))
+        {
+            myTarget.BlockNode();
+        }
     }
 }
```

Comment:
end of file line please

Reply:
La ligne est bien là dans le fichier original